csrankings-k.csv

```diff
@@ -114,7 +114,6 @@ Kameswari Chebrolu,IIT Bombay,https://www.cse.iitb.ac.in/~chebrolu,U-Irc6kAAAAJ
 Kami E. Vaniea,University of Edinburgh,https://vaniea.com,G_l2kiQAAAAJ
 Kami Vaniea,University of Edinburgh,https://vaniea.com,G_l2kiQAAAAJ
 Kamil Saraç,University of Texas at Dallas,http://www.utdallas.edu/~ksarac,NOSCHOLARPAGE
-Kamin Whitehouse,University of Virginia,http://www.cs.virginia.edu/~whitehouse,ob_3jn4AAAAJ

```

Comment:
Please move entry to `old/industry.csv`.

Reply:
Done